src/index.js

```diff
@@ -0,0 +1,36 @@
+// import _ from 'lodash';
+import './style.css';
+
+const list = document.querySelector('.list-container');
+
+class Task {
+  constructor(description = '', completed = false, index = 0) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const tasks = [];
+
+function addTasks(description, completed, index) {
+  const demoTask = new Task(description, completed, index);
+  tasks.push(demoTask);
+}
+
+addTasks('Demo task 1', false, 1);
+addTasks('Demo task 2', false, 2);
+
+function displaytasks() {
+  let taskgenerator = '';
+  tasks.forEach((task) => {
```

Comment:
- [ ] Kindly sort the list based on the index value before displaying it as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.).